src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="OMIA:002065"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0011873 {source="OMIA:002065"} ! cross-species analog Niemann-Pick disease, type C2
+
+[Term]
+id: MONDO:1010189
+name: neuronal ceroid lipofuscinosis, 3, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 3 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002432 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002432"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002432"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008767 {source="OMIA:002432"} ! cross-species analog neuronal ceroid lipofuscinosis 3
+
+[Term]
+id: MONDO:1010190
+name: Gaucher disease, type II, non-human animal
+def: "Gaucher disease, type II that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002621 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002621"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002621"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009266 {source="OMIA:002621"} ! cross-species analog Gaucher disease type II
+
+[Term]
+id: MONDO:1010191
+name: deficiency of uridine monophosphate synthase, non-human animal
+def: "Deficiency of uridine monophosphate synthase that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000262 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000262"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000262"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009797 {source="OMIA:000262"} ! cross-species analog orotic aciduria
+
+[Term]
+id: MONDO:1010192
+name: central core myopathy, non-human animal
+def: "Central core myopathy that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000172 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000172"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0007294 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog central core myopathy
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:1010194
+name: muscular hypertrophy (double muscling), non-human animal
+def: "Muscular hypertrophy (double muscling) that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000683 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000683"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000683"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0013598 {source="OMIA:000683"} ! cross-species analog myostatin-related muscle hypertrophy
```

Comment:
is this correct? or should it be a less specific, not gene-related disease?

Reply:
This is correct, but I the cross-species analog should be at the animal species level, not the species non-specific level.